client/components/examples/activity/NewActivity.vue

```diff
@@ -41,19 +46,52 @@
         </el-col>
       </el-row>
       <el-row type="flex" justify="flex-start">
-        <el-col :xs="24" :sm="10">
-          <el-form-item :label="$t('activity.type')" prop="type">
+        <el-col
+          :xs="24"
+          :sm="10"
+        >
+          <el-form-item
+            :label="$t('activity.type.label')"
+            prop="type"
+          >
             <el-checkbox-group v-model="formData.type">
-              <el-checkbox :label="$t('activity.price')"></el-checkbox>
-              <el-checkbox :label="$t('activity.rights')"></el-checkbox>
+              <el-checkbox
+                label="activity.type.price"
+              >
+                {{$t('activity.type.price')}}
+              </el-checkbox>
+              <el-checkbox
+                label="activity.type.rights"
+              >
+                {{$t('activity.type.rights')}}
+              </el-checkbox>
             </el-checkbox-group>
           </el-form-item>
         </el-col>
-        <el-col :xs="24" :sm="{span: 10, offset: 2}">
-          <el-form-item :label="$t('activity.priority')" prop="priority">
+        <el-col
+          :xs="24"
+          :sm="{span: 10, offset: 2}"
+        >
+          <el-form-item
+            :label="$t('activity.priority.label')"
+            prop="priority"
+          >
             <el-radio-group v-model="formData.priority">
-              <el-radio :label="$t('activity.medium')"></el-radio>
-              <el-radio :label="$t('activity.high')"></el-radio>
+              <el-radio
+                label="activity.priority.low"
+              >
+                {{$t('activity.priority.low')}}
```

Comment:
Element UI uses label for both value and label.
At least it allows using default slot for i18n.